canonical-web-and-design/react-components
# Setup project for TypeScript and migrate Tooltip component.

## Done
* Setup TypeScript
* Add eslint config
* Add tsconfig with declaration support
* Migrate `Tooltip` to TypeScript.

## QA
* Ensure project builds
* Ensure storybook runs as expected

Fixes: #176
